sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -513,15 +517,21 @@ func (host *nodeLanguageHost) execNodejs(
 			env = append(env, "PULUMI_NODEJS_TSCONFIG_PATH="+host.tsconfigpath)
 		}
 
+		var nodeargs []string
+		if host.nodeargs != "" {
+			nodeargs = strings.Split(host.nodeargs, " ")
```

Comment:
Could be an issue if any args use space inside quotes or such, but presumably that'd rarely be the case.

Reply:
Good point.  Probably unlikely - but I've pulled in https://pkg.go.dev/github.com/google/shlex to ensure this works correctly.